tests/testlayers/mountain_peak/layer.sql

```diff
@@ -0,0 +1,62 @@
+-- etldoc: layer_mountain_peak[shape=record fillcolor=lightpink,
+-- etldoc:     style="rounded,filled", label="layer_mountain_peak | <z7_> z7+" ] ;
+
+CREATE OR REPLACE FUNCTION layer_mountain_peak(bbox geometry,
```

Comment:
do you actually need this file as part of the test? I think you might as well just delete it -- doesn't appear like you are testing for any of its functionality. Or make it empty, etc.  The shorter the test, the easier it will be to debug issues that it could trigger.